mast/config.py

```diff
@@ -15,3 +15,5 @@ class Config:
     MAIL_USE_TLS = True
     MAIL_USERNAME = os.getenv('EMAIL_USER')
     MAIL_PASSWORD = os.getenv('EMAIL_PASS')
+    #TODO: not working on local with this field
+    #SERVER_NAME = 'mathletics-test.ks.matfyz.cz'

```

Comment:
Uncomment #SERVER_NAME

Reply:
When uncommented some things break on local, it is necessary to have it on test-server afaik.